composer.json

```diff
@@ -12,12 +12,12 @@
         }
     ],
     "require": {
-        "illuminate/mail": "^7.0",
-        "guzzlehttp/guzzle": "^6.3",
+        "guzzlehttp/guzzle": "^7.1",
+        "illuminate/mail": "^8.7",
         "sendinblue/api-v3-sdk": "^6.1"
```

Comment:
I think you should upgrade this too to ^7.0 as there's a newer version

Reply:
If I remember well they changed the transaction mail sending API. I'll check it and modify the code if needed